source/declarations.tex

```diff
@@ -2921,7 +2921,7 @@
 Because a \grammarterm{using-declaration} designates a base class member
 (and not a member subobject or a member function of a base class
 subobject), a \grammarterm{using-declaration} cannot be used to resolve
-inherited member ambiguities. For example,
+inherited member ambiguities by itself. For example,
```

Comment:
This seems to be suggesting that the using-declaration may help when combined with something else. Is that actually true? Can we extend the example to show that, instead?